lib/MusicBrainz/Server/Controller/Role/WikipediaExtract.pm

```diff
@@ -46,8 +52,14 @@ sub _get_extract
         } @{ $entity->relationships_by_link_type_names('wikipedia', 'wikidata') };
 
     if (scalar @links) {
+        my $editor;
+        if ($c->user_exists) {
+            $editor = $c->model('Editor')->get_by_id($c->user->id);
+            $c->model('EditorLanguage')->load_for_editor($editor);
```

Comment:
You should be able to just call `load_for_editor` on `$c->user`, and pass `$c->user` below.

Reply:
Done in https://github.com/metabrainz/musicbrainz-server/pull/633/commits/8622f1311f9d796811ec83c00f135e1174747d89